lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -122,7 +121,8 @@ def test_credit_requirements_eligible(self):
             response,
             "{}, you have met the requirements for credit in this course.".format(self.USER_FULL_NAME)
         )
-        self.assertContains(response, "Completed by {date}".format(date=self._now_formatted_date()))
+        self.assertContains(response, "Completed by {date}")
+        self.assertContains(response, datetime.datetime.now(UTC).strftime('%Y-%m-%d %H:%M'))
```

Comment:
^^Hour resolution seems to still pass the tests and satisfy the spirit, and though it's far from a perfect solution, I think it works.